https://src.fedoraproject.org/rpms/subversion/raw/rawhide/f/subversion-1.14.1-python-3.11-build.patch

Use read_file instead of deprecated readfp  - removed in Python 3.11

--- a/build/generator/gen_base.py
+++ b/build/generator/gen_base.py
@@ -76,7 +76,7 @@
 
     # Now read and parse build.conf
     parser = configparser.ConfigParser()
-    parser.readfp(open(fname))
+    parser.read_file(open(fname))
 
     self.conf = build_path(os.path.abspath(fname))
 

